zulip_bots/setup.py

```diff
@@ -36,7 +46,7 @@
 
 setuptools_info = dict(
     install_requires=[
-        'zulip>=0.3.4',
+        'zulip>={}'.format(ZULIP_BOTS_VERSION),
```

Comment:
From my experience as a maintainer so far, I have never had to make separate releases for these packages with separate version numbers. For now, I think we should stick with one version for all packages, since they will be released simultaneously (ideally). Actually, I wanted to have a universal `version.py` in the repo root directory, but the way packaging works, that wouldn't have worked!

I don't think we should over-complicate this by writing a script that tries to get the version number from an adjoining package, that wouldn't work anyway because each package is its own distinct self (so to speak) and shouldn't depend on anything outside of it *before* it has been installed.

I do agree with you on one thing that they are technically independent, so let's just skip the version and say `zulip` instead of `zulip>=<version>`. The former invocation would install the latest package automatically, so we don't need to explicitly specify it here. :) Thanks for pointing this out, Robert!

Reply:
My reasoning was that eventually, some package like `zulip_botserver` might be very stable and doesn't need to be updated, whereas frequent updates occur for other packages. In such a case, I think it'd be more convenient for users of the packages to be able to not worry about three different package version when they are in fact all the same. Maybe we can discuss this at today's meeting :)

Reply:
Okay, I emphasized this in a comment further down. I agree with you on that. I just don't think that separation is something that needs to be done as a part of this PR. That could probably be done later on when we realize that the packages are diverging in terms of stability and versioning scheme. :)